client/a8c-for-agencies/sections/signup/signup-v2/components/multi-step-form/contact-form/index.tsx

```diff
@@ -0,0 +1,167 @@
+import './style.scss';
+
+type FormData = {
+	firstName: string;
+	lastName: string;
+	email: string;
+	agencyName: string;
+	businessUrl: string;
+	phoneNumber: string;
+};
+
+type Props = {
+	onContinue: () => void;
+};
+
+const SignupContactForm = ( { onContinue }: Props ) => {
+	const translate = useTranslate();
+
+	const countriesList = useGetSupportedSMSCountries();
+	const noCountryList = countriesList.length === 0;
+
+	const [ formData, setFormData ] = useState< FormData >( {
+		firstName: '',
+		lastName: '',
+		email: '',
+		agencyName: '',
+		businessUrl: '',
+		phoneNumber: '',
+	} );
+
+	const handlePhoneInputChange = ( data: { phoneNumberFull: string } ) => {
+		setFormData( ( prev ) => ( {
+			...prev,
+			phoneNumber: data.phoneNumberFull,
+		} ) );
+	};
+
+	const handleInputChange =
+		( field: keyof FormData ) => ( event: React.ChangeEvent< HTMLInputElement > ) => {
+			setFormData( ( prev ) => ( {
+				...prev,
+				[ field ]: event.target.value,
+			} ) );
+		};
+
+	const handleSubmit = ( e: React.FormEvent ) => {
+		e.preventDefault();
+		onContinue();
+	};
+
+	return (
+		<Form
+			className="signup-contact-form"
+			title={ translate( 'Sign up' ) }
+			description={ translate(
+				'Join 5000+ agencies and grow your business with Automattic for Agencies. It only takes two minutes!'
+			) }
+		>
+			<div className="signup-multi-step-form__fields">
+				<div className="signup-multi-step-form__name-fields">
+					<FormField label={ translate( 'Your first name' ) } isRequired>
+						<FormTextInput
+							name="firstName"
+							value={ formData.firstName }
+							onChange={ handleInputChange( 'firstName' ) }
+						/>
+					</FormField>
+
+					<FormField label={ translate( 'Last name' ) } isRequired>
+						<FormTextInput
+							name="lastName"
+							value={ formData.lastName }
+							onChange={ handleInputChange( 'lastName' ) }
+						/>
+					</FormField>
+				</div>
+
+				<FormField label={ translate( 'Email' ) } isRequired>
+					<FormTextInput
+						name="email"
+						type="email"
+						value={ formData.email }
+						onChange={ handleInputChange( 'email' ) }
+					/>
+				</FormField>
+
+				<FormField label={ translate( 'Agency name' ) } isRequired>
+					<FormTextInput
+						name="agencyName"
+						value={ formData.agencyName }
+						onChange={ handleInputChange( 'agencyName' ) }
+					/>
+				</FormField>
+
+				<FormField label={ translate( 'Business URL' ) } isRequired>
+					<FormTextInput
+						name="businessUrl"
+						value={ formData.businessUrl }
+						onChange={ handleInputChange( 'businessUrl' ) }
+					/>
+				</FormField>
+
+				{ noCountryList && <QuerySmsCountries /> }
+
+				<FormField label={ translate( 'Phone number' ) } showOptionalLabel>
+					<FormPhoneInput
+						countrySelectProps={ {
+							'data-testid': 'a4a-signup-country-code-select',
+						} }
+						phoneInputProps={ {
+							'data-testid': 'a4a-signup-phone-number-input',
+						} }
```

Comment:
We haven't added any test cases, and we already have the same `data-testid`s in the original signup form. Can we remove this for now or keep it different?